code/modules/mob/living/carbon/human/species_types/synths.dm

```diff
@@ -39,13 +41,14 @@
 	. = ..()
 	UnregisterSignal(H, COMSIG_MOB_SAY)
 
-/datum/species/synth/handle_chemicals(datum/reagent/chem, mob/living/carbon/human/H)
-	if(chem.type == /datum/reagent/medicine/synthflesh)
-		chem.reaction_mob(H, TOUCH, 2 ,0) //heal a little
-		H.reagents.remove_reagent(chem.type, REAGENTS_METABOLISM)
-		return 1
+/datum/species/synth/handle_reagents(mob/living/carbon/human/H, datum/reagent/R)
+	if(istype(R, /datum/reagent/medicine/synthflesh))
```

Comment:
Synthflesh is not supposed to do anything if not applied via patch, you should check that first.

Reply:
for synths it purposly does